docs/dev/features/netapp-mounting/overview.md

```diff
@@ -0,0 +1,80 @@
+This controller is built off of the existing `aaw-kubeflow-profiles-controller` and is simple. It uses the same built image that the other controllers use, with the exception that the startup command is `namespace` to run this controller. The tracking issue for this controller is BTIS-332 with more details.
+All this does is watch profiles, and  then for that profile's generated namespace, if it does not have the label listed in `newLabels`, then in adds it. In this case it adds `filer-sidecar-injection: enabled`.
+
+### Deployment Checklist
+- [x] Make changes to the aaw-kubeflow-profiles-controller repo on github and push and let the workflow push the image
+- [x] Modify the `profiles-controller` branch in the `statcan/charts` gitlab repository, modifying any new permissions needed, updating the tag, or adding a new controller
+- [x] Modify the `profiles-controller.yaml` tag in the argocd-manifests repo.
+- [x] Sync the respective `root` application in argocd, for example for prod you need to sync the `das-prod-cc-00-root` application for everything to come up.
+
+### [Ontap CVO controller](https://github.com/StatCan/aaw-kubeflow-profiles-controller/blob/profiles-controller-aaw2.0/cmd/ontap-cvo.go)
+Thie is a **WIP**, with its current purpose being to generate a `filer-conn-secret` based on values in a configmap. This is a WIP as we currently do not have the ability to create an account with the permissions necessary to create the account.
+
+This controller relies on the existence of a secret that has the following information;
+- the User's username as it exists on AVD, this needs to match up exactly. For example for jose-matsuda it would be `matsujo`. This is due to how the permissions themselves are mapped.
+- the bucket or share that the user needs access to. This is required so that when we create the secret, it has the information on what level, and the mounting solution requires the bucket name.
+- The name of the filer that the bucket or share is on. This will be used to populate the secret, as the mounting solution requires the S3 url. 
+
+This works like all the other profiles controllers, and steps can be taken from above as well. The controller watches profile resources and will generate a unique to bucket secret (for as many filers as a user has acccess to there should be a secret) following the naming convention of `*filer-conn-secret` for the profile that contains the following fields
+- S3_ACCESS
+- S3_BUCKET
+- S3_SECRET
+- S3_URL
+
+[This is the API we are trying to call, but do not have permissions to](https://docs.netapp.com/us-en/ontap-restapi/ontap/protocols_s3_services_svm.uuid_users_endpoint_overview.html#creating-an-s3-user-configuration)
+
+**Again, this is a work in progress.**
+It will need to change based on work done by Mathis when users select what filer (and level of) that they have access to.

```

Comment:
Not just saying this because my name is there ;) Should we instead say "... on work done in the "Manage Filers" page in central dashboard where users select ..." , to not refer to an individual just in case this Mathis guy is abscent that day, or not on the team anymore or whatever.